### ISSUE TITLE
Preis wird nach einiger Zeit nicht mehr aktualisiert

### ISSUE THREAD

**Issue by viper-666** [Codibris/ioBroker.tibberconnect#209]
Ich habe bei mir das Problem, dass der Adapter einige Zeit problemlos funktioniert. Aber irgendwann wird der Preis wohl nicht mehr von Tibber abgerufen und aktualisiert.
Der Adapter ist aber grün, wenn ich den neu starte wird der Preis wieder für einige Zeit aktualisiert bis der Fehler wieder auftritt.
Im Log ist nichts zu finden.

**Comment by kopierschnitte** [comment]
Ja, bei mir auch. Lasse die Instanz jetzt alle 3 Tage neustarten. Hilft erstmal.

**Comment by MubiTec** [comment]
Bei mir das gleiche, dann mach ich wohl auch erstmal den workaround
